graphql.go

```diff
@@ -48,25 +48,26 @@ import (
 	"github.com/pkg/errors"
 )
 
+type Client interface {
```

Comment:
This change caused a problem in my repo where I was using a * to graphql.Client...!!

Reply:
Same issue with edge-output-writer https://github.com/veritone/edge-output-writer/blob/master/api/graphql.go#L78 where I was using

Reply:
The clients of graphql need to be updated to use the new API.